courses/models.py

```diff
@@ -0,0 +1,52 @@
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+from django.db import models
+from jsonfield import JSONField
+from uuidfield import UUIDField
+
+class Course(models.Model):
+    """
+    An edX course which is purchaseable.
+    """
+    uuid = UUIDField(auto=True)
+    title = models.CharField(max_length=255)
+    author_name = models.CharField(max_length=255)
+    overview = models.TextField()
+    description = models.TextField()
+    video_url = models.URLField()
+
+    edx_instance = models.CharField(max_length=255)
+    instructors = models.ManyToManyField(User)
+    live = models.BooleanField(default=False)
+
+    price_per_seat_cents = models.IntegerField(null=True, blank=True, help_text="Cost of the whole course per seat in cents")
+
+    deleted = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True, blank=True)
+    updated_at = models.DateTimeField(auto_now=True, blank=True)
+
+    def __unicode__(self):
+        return self.title
+
+    @property
+    def url(self):
+        return reverse('course-detail', args=(self.uuid,))
```

Comment:
These links don't include the `http://host/` part of the URL which makes them different from the ones generated by `HyperlinkedModelSerializer`. Is it possible to generate the prefix for this URL to make them consistent? If it's messy or complicated to do, feel free to ignore, it's obviously functional as is